tests/test_drivers.py

```diff
@@ -49,5 +52,41 @@ def testMissingSubmitPath(self):
             _init_submission_driver({"payload": {"outputRun": "bad"}})
 
 
+class TestPingDriver(unittest.TestCase):
+    def testWmsServiceSuccess(self):
+        retval = ping_driver("wms_test_utils.WmsServiceSuccess")
+        self.assertEqual(retval, 0)
+
+    def testWmsServiceFailure(self):
+        with self.assertLogs(level=logging.INFO) as cm:
```

Comment:
You can be more specific with

```python
with self.assertLogs("lsst.ctrl.bps", level="ERROR"):
```
since you know that an ERROR log must be issued.